### PR TITLE
NEW! RoomView pt1.

### PR DESCRIPTION
**Pull request by rauchp**
- big dict energy
- folder consistency be like 
- started chat view
- fixed flair screen flatlist
- took renderFlair and renderAvatar out of single screen and into shared areas
- right now we have dict and obj objects for avatar + team logos. we could switch to dynamically making lists later from dicts, so we don't have to be updating in two places.

### CODE REVIEW

File: civilapp/current/RoomView.js [hunk 1]
```diff
@@ -261,100 +280,109 @@ class RoomView extends React.Component {
     }
   }
 
-  subscribeToBroadcasterEvents = roomId => {
```
Comment (rauchp) [inline civilapp/current/RoomView.js]: this'll be back soon™

File: civilapp/auth/AuthLoadingScreen.js [hunk 2]
```diff
@@ -73,6 +73,7 @@ export default class AuthLoadingScreen extends Component {
       const userId = await initializeUserData()
 
       this.props.navigation.navigate('Landing', { userId })
+    //  this.props.navigation.navigate('Current', { roomId: '9b2463c1-4fd1-4325-933f-4de1a12b6013' })
```
Comment (rauchp) [inline civilapp/auth/AuthLoadingScreen.js]: using this rn for testing, will remove

File: civilapp/browse/common/SportsCard.js [hunk 3]
```diff
@@ -105,32 +101,11 @@ class SportsCard extends Component {
...
-    } else {
-      console.log('error: could not find avatar in dictionary')
-      return
-    }
-  }
+
   renderUpperHalf = gameIsOn => {
     const { size, extraTags } = this.state
     const speakers = this.props.speakers
+    console.log('WHAT:?', this.props)
```
Comment (carteryu) [inline civilapp/browse/common/SportsCard.js]: xD

File: civilapp/components/SuperText.js [hunk 4]
```diff
@@ -99,11 +99,13 @@ const RetroBrowseCardText = (props) => {
             fontSize = 8
         multiplier = 1.75
     }
-
-    const shadowTextStyle = { zIndex: -1, position: 'absolute', color: 'black', fontFamily, fontSize }
+    if(props.color)
+        color = props.color
+    const shadowTextStyleInit = { zIndex: -1, position: 'absolute', color: 'black', fontFamily, fontSize } 
+    const shadowTextStyle =  { ...shadowTextStyleInit, ...props.style}
```
Comment (carteryu) [inline civilapp/components/SuperText.js]: sry brainlet here - this just merges shadowtextstyleinit and props.style objects right?

**Pull request merged**